src/util/raindropTransformer.ts

```diff
@@ -70,10 +70,7 @@ type RaindropResponse = {
 };
 
 // below are normalized values
-export const raindropTransformer = (
-  r: RaindropResponse,
-  collections: Collection[]
```

Comment:
`collections` was unused, removed it to not need to pass something when transforming the api-response. Might not be the best solution, depending on what you originally intended to do with the parameter

Reply:
No downsides in removing it — if we want it later, we can add it back in. It's a good reminder to set up some better linting for catching unused values